src/ResourceManager/PowerBIEmbedded/ChangeLog.md

```diff
@@ -19,36 +19,19 @@
 -->
 ## Current Release
 
-## Version 4.0.0
```

Comment:
@selasagi please revert the deletions made to this file - the new content you are adding can remain under the `## Current Release` header

Reply:
done